src/components/Sidebar/PostDetails/PostDetails.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from 'react';
+import { NewCommentForm } from './NewCommentForm';
+import {
+  ActivePostContext,
+  CommentFormContext,
+  CommentListContext,
+} from '../../../utils/Store';
+import { CommentList } from './../CommentList/CommentList';
+
+export const PostDetails: React.FC = () => {
+  const { activePost } = useContext(ActivePostContext);
+  const { comments } = useContext(CommentListContext);
+  const { isActiveForm } = useContext(CommentFormContext);
+
+  return (
+    <div className="content" data-cy="PostDetails">
+      <div className="content" data-cy="PostDetails">
```

Comment:
It seems like there is a duplicate 'div' with the same className and data-cy attribute. Consider removing one of them to avoid redundancy.